### PR TITLE
GH actions: added auth odbc{sqlitle3, mssql} tests. Removed from CircleCI

### PR DESCRIPTION
**Pull request by romeroalx**
### Short description
Added auth regression tests for odbc-sqlitle3 and odbc-mssql to GitHub Actions. Removed from CircleCI.

### Checklist

I have:
- [ ] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [x] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: tasks.py
Comment (fredmorcos) [inline tasks.py:570]: Feel free to align these with the table above them.

File: tasks.py
Comment (fredmorcos) [inline tasks.py:593]: I would prefer to use python for something like this. As an example putting the file contents in a variable called `odbc_config` and doing something like the following:

```python
with open("~/.odbc.ini", "w") as f:
    print(odbc_config, file=f)
```

File: tasks.py
Comment (fredmorcos) [inline tasks.py:608]: Same here.

**Review by fredmorcos — CHANGES_REQUESTED** [review]

File: tasks.py
Comment (fredmorcos) [reply tasks.py:593]: Or of course using `f.write(...)`

Comment (romeroalx) [reply tasks.py]: Done

File: tasks.py
Comment (romeroalx) [reply tasks.py]: Done

File: tasks.py
Comment (fredmorcos) [inline tasks.py:610]: Perhaps use a [triple quoted string](https://docs.python.org/3/library/stdtypes.html?highlight=triple%20quoted%20string#text-sequence-type-str) here?

**Review by fredmorcos — APPROVED** [review]
Other than this one comment everything looks fine to me. However I would still recommend waiting for another review from @Habbie

File: tasks.py
Comment (Habbie) [inline tasks.py:639]: I'm not sure this comment makes sense here any more

**Review by Habbie — CHANGES_REQUESTED** [review]
noted one possibly spurious comment. Fred's comment makes sense. Otherwise, great work!

File: tasks.py
Comment (romeroalx) [reply tasks.py:639]: You are correct! I will clean that up now.

File: tasks.py
Comment (Habbie) [reply tasks.py:610]: how about this one?

Comment (romeroalx) [reply tasks.py:610]: Yes, I noticed that one an hour ago. Running tests against my fork. I will update this PR once confirmed I do not break anything

**Review by Habbie — APPROVED** [review]

**Pull request merged**
